<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHOW IMAGE GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>231</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>231</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="actionBrightness"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegative"/>
    <addaction name="actionBiner"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionHistogram_Grayscale"/>
    <addaction name="actionHistogram_RGB"/>
    <addaction name="actionHistogram_Equalization"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <addaction name="actionTranslation"/>
    <addaction name="actionRotation"/>
    <addaction name="actionResize"/>
    <addaction name="actionCrop"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Aritmatika">
    <property name="title">
     <string>Operasi Aritmatika</string>
    </property>
    <addaction name="actionTambah_dan_Kurang"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Boolean">
    <property name="title">
     <string>Operasi Boolean</string>
    </property>
    <addaction name="actionOperasi_AND"/>
    <addaction name="actionOperasi_OR"/>
    <addaction name="actionOperasi_XOR"/>
   </widget>
   <widget class="QMenu" name="menuKonvolusi">
    <property name="title">
     <string>Konvolusi</string>
    </property>
    <widget class="QMenu" name="menuKonvolusi_2D">
     <property name="title">
      <string>Konvolusi 2D</string>
     </property>
     <addaction name="actionKonvolusi_A"/>
     <addaction name="actionKonvolusi_B"/>
    </widget>
    <widget class="QMenu" name="menuD2">
     <property name="title">
      <string>D2</string>
     </property>
     <addaction name="actionKonvolusi_i"/>
     <addaction name="actionKonvolusi_ii"/>
    </widget>
    <widget class="QMenu" name="menuSharpener">
     <property name="title">
      <string>Sharpener</string>
     </property>
     <addaction name="actionSharp_i"/>
     <addaction name="actionSharp_ii"/>
     <addaction name="actionSharp_iii"/>
     <addaction name="actionSharp_iv"/>
     <addaction name="actionSharp_v"/>
     <addaction name="actionSharp_vi"/>
    </widget>
    <addaction name="menuKonvolusi_2D"/>
    <addaction name="menuD2"/>
    <addaction name="actionGaussian"/>
    <addaction name="menuSharpener"/>
    <addaction name="actionLaplace"/>
    <addaction name="actionMedian"/>
    <addaction name="actionMax"/>
   </widget>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuHistogram"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuOperasi_Aritmatika"/>
   <addaction name="menuOperasi_Boolean"/>
   <addaction name="menuKonvolusi"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Streching</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionBiner">
   <property name="text">
    <string>Biner</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionTranslation">
   <property name="text">
    <string>Translation</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="text">
    <string>Rotation</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionTambah_dan_Kurang">
   <property name="text">
    <string>Tambah dan Kurang</string>
   </property>
  </action>
  <action name="actionOperasi_AND">
   <property name="text">
    <string>Operasi AND</string>
   </property>
  </action>
  <action name="actionOperasi_OR">
   <property name="text">
    <string>Operasi OR</string>
   </property>
  </action>
  <action name="actionOperasi_XOR">
   <property name="text">
    <string>Operasi XOR</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionPelembutan">
   <property name="text">
    <string>Pelembutan</string>
   </property>
  </action>
  <action name="actionKonvolusi_A">
   <property name="text">
    <string>Konvulasi A</string>
   </property>
  </action>
  <action name="actionKonvolusi_B">
   <property name="text">
    <string>Konvolusi B</string>
   </property>
  </action>
  <action name="actionKonvolusi_i">
   <property name="text">
    <string>Konvulasi i</string>
   </property>
  </action>
  <action name="actionKonvolusi_ii">
   <property name="text">
    <string>Konvolusi ii</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Gaussian</string>
   </property>
  </action>
  <action name="actionSharp_i">
   <property name="text">
    <string>Sharp i</string>
   </property>
  </action>
  <action name="actionSharp_ii">
   <property name="text">
    <string>Sharp ii</string>
   </property>
  </action>
  <action name="actionSharp_iii">
   <property name="text">
    <string>Sharp iii</string>
   </property>
  </action>
  <action name="actionSharp_iv">
   <property name="text">
    <string>Sharp iv</string>
   </property>
  </action>
  <action name="actionSharp_v">
   <property name="text">
    <string>Sharp v</string>
   </property>
  </action>
  <action name="actionSharp_vi">
   <property name="text">
    <string>Sharp vi</string>
   </property>
  </action>
  <action name="actionLaplace">
   <property name="text">
    <string>Laplace</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="actionMax">
   <property name="text">
    <string>Max</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
